invenio/modules/search/walkers/elasticsearch.py

```diff
@@ -38,6 +42,27 @@ class ElasticSearchDSL(object):
 
     # pylint: disable=W0613,E0102
 
+    def __init__(self):
+        """Provide a dictinary mapping invenio keywords
+        to elasticsearch fields as a list
+        eg. {"author": ["author.last_name, author.first_name"]}
+        """
+        # FIXME Configure the dictionary for mapping
+        self.keyword_dict = None
```

Comment:
Can you create some configuration variable like: `SEARCH_ELASTIC_KEYWORD_MAPPING`?

--
Usage:
```python
self.keyword_mapping = cfg['...']
```